### PR TITLE
Refactor iOS Deeplink Manager

### PR DESCRIPTION
**Pull request by vinothvino42**

### CODE REVIEW

File: lib/ios_deep_link_manager.dart [hunk 1]
```diff
@@ -3,7 +3,7 @@ import 'package:flutter/services.dart';
 
 class IOSDeepLinkManager extends DeepLinkNavigator {
   static final IOSDeepLinkManager _instance = IOSDeepLinkManager._internal();
-  static const MethodChannel _platform = MethodChannel('com.tin.mobile');
+  static const MethodChannel _platform = MethodChannel('com.ensemble.platform');
```
Comment (vusters) [inline lib/ios_deep_link_manager.dart]: com.ensembleui - that's our domain (not com.ensemble)

**Review by vusters — CHANGES_REQUESTED** [review]
Please review the regular deep_link_manager. That was changed to pass the arguments to the next screen. Please make sure to do the same for iOS-specific

File: lib/ios_deep_link_manager.dart (hunk 1 above)
Comment (vinothvino42) [reply lib/ios_deep_link_manager.dart]: I already updated it to ```com.ensembleui.host.platform```

Comment (vinothvino42) [reply lib/ios_deep_link_manager.dart]: Added in the starter project. 
PR - https://github.com/EnsembleUI/starter/pull/33

**Review by vusters — APPROVED** [review]

**Pull request merged**